incubator/vault/values.yaml

```diff
@@ -100,15 +100,6 @@ vault:
   # https://www.vaultproject.io/intro/getting-started/dev-server.html for more
   # information.
   dev: true
-  # Allows the mounting of various custom secrets th enable production vault
-  # configurations. The comments show an example usage for mounting a TLS
-  # secret. The two fields required are a secretName indicating the name of
-  # the Kubernetes secret (created outside of this chart), and the mountPath
-  # at which it should be mounted in the Vault container.
-  customSecrets: []
-    # - secretName: vault-tls
-    #   mountPath: /vault/tls
-  #
```

Comment:
This looks like a breaking change as it's removing existing fields. Is that we could keep the existing field otherwise I'm not sure this is a patch level change and it might constitute a minor version bump.

Reply:
A [previous iteration of this PR](https://github.com/helm/charts/pull/9250/commits/f4a66018f4ef354d81d5201ae3cac36f40bf1146) added extraVolumeMounts without removing customSecrets. I can revert to that commit if you think it would stand a better chance of getting merged.

Reply:
I think this has implications for rolling back to a previous chart version. I would like to see what the chart maintainer would like to do here. cc @jpds

Reply:
It might be okay to bump the minor version in this case